vendor/init.go

```diff
@@ -66,22 +73,68 @@ var files = []file{
 	},
 }
 
+func addChrome(ctx context.Context) error {
+	// Bucket URL: https://console.cloud.google.com/storage/browser/chromium-browser-continuous/?pli=1
+	storageBktName := "chromium-browser-continuous"
+	prefixLinux64 := "Linux_x64"
+	lastChangeFile := "Linux_x64/LAST_CHANGE"
+	chromeFilename := "chrome-linux.zip"
+	gcsPath := fmt.Sprintf("gs://%s/", storageBktName)
+	client, err := storage.NewClient(ctx, option.WithHTTPClient(http.DefaultClient))
+	if err != nil {
+		return fmt.Errorf("cannot create a storage client for downloading the chrome browser: %s", err)
+	}
+	bkt := client.Bucket(storageBktName)
+	obj := bkt.Object(lastChangeFile)
+	r, err := obj.NewReader(ctx)
+	if err != nil {
+		return fmt.Errorf("cannot create a reader for %s%s file: %s", gcsPath, lastChangeFile, err)
+	}
+	defer r.Close()
+	// Read the last change file content for the latest build directory name
+	data, err := ioutil.ReadAll(r)
+	if err != nil {
+		return fmt.Errorf("cannot read from %s%s file: %s", gcsPath, lastChangeFile, err)
+	}
+	latestChromeBuild := string(data)
+	latestChromePackage := path.Join(prefixLinux64, latestChromeBuild, chromeFilename)
+	chromeObjHandler := bkt.Object(latestChromePackage)
+	cpAttrs, err := chromeObjHandler.Attrs(ctx)
+	if err != nil {
+		return fmt.Errorf("cannot get the chrome package %s%s attrs: %s", gcsPath, latestChromePackage, err)
+	}
+	files = append(files, file{
+		name:     chromeFilename,
+		browser:  true,
+		hash:     hex.EncodeToString(cpAttrs.MD5),
+		hashType: "md5",
+		url:      cpAttrs.MediaLink,
+	})
+	return nil
+}
+
 func main() {
 	flag.Parse()
-
+	ctx := context.Background()
+	if *downloadBrowsers {
+		if err := addChrome(ctx); err != nil {
+			glog.Errorf("Unable to Download Google Chrome browser. Continuing...")
```

Comment:
Add the error to the message. (This was a previous review comment, see there for more detail).